doc/user-manual/language/cubical.lagda.rst

```diff
@@ -1,82 +1,145 @@
 
--------------------------
-The interval type (``I``)
--------------------------
+  open import Cubical.Core.Everything
 
-The variable ``i`` in the definition of ``refl`` has type ``I``, which
-topologically corresponds to the `real unit interval <https://en.wikipedia.org/wiki/Unit_interval>`_.
-In a closed context, there are only two values of type ``I``: the two
-ends of the interval, ``i0`` and ``i1``::
+For detailed install instructions for ``agda/cubical`` see:
+https://github.com/agda/cubical/blob/master/INSTALL.md. In order to
+make this library visible to Agda add
+``/path/to/cubical/cubical.agda-lib`` to ``.agda/libraries`` and
+``cubical`` to ``.agda/defaults`` (where ``path/to`` is the absolute
+path to where the ``agda/cubical`` library has been installed). For
+details of Agda's library management see :ref:`package-system`.
 
-  a b : I
-  a = i0
-  b = i1
+Expert users who doesn't want to rely on ``agda/cubical`` can just add
+the relevant import statements at the top of their file (for details
+for experts see :ref:`primitives-ref`). However, for beginners it is
+recommended that one uses at least the core part of the
+``agda/cubical`` library.
 
-Elements of the interval form a `De Morgan algebra <https://en.wikipedia.org/wiki/De_Morgan_algebra>`_,
-with minimum (``∧``), maximum (``∨``) and negation (``~``).
+There is also an older version of the library available at
+https://github.com/Saizan/cubical-demo/. However this is relying on
+deprecated features and is not recommended to use.
 
-.. code-block:: agda
+The interval and path types
+---------------------------
 
-  module interval-example₁ (i j : I) where
-    data _≡_ (i : I) : I → Set where
-      reflI : i ≡ i
+The key idea of Cubical Type Theory is to add an interval type ``I :
+Setω`` (the reason this is in ``Setω`` is because it doesn't support
+the ``transp`` and ``hcomp`` operations). A variable ``i : I``
+intuitively corresponds to a point the `real unit interval
+<https://en.wikipedia.org/wiki/Unit_interval>`_. In an empty context,
+there are only two values of type ``I``: the two endpoints of the
+interval, ``i0`` and ``i1``.
 
-    infix 10 _≡_
+.. code-block:: agda
 
-    max min neg : I
+  i0 : I
+  i1 : I
+
+Elements of the interval form a `De Morgan algebra
+<https://en.wikipedia.org/wiki/De_Morgan_algebra>`_, with minimum
+(``∧``), maximum (``∨``) and negation (``~``).
 
 .. code-block:: agda
 
-    max = i ∨ j
-    min = i ∧ j
-    neg = ~ i
+  _∧_ : I → I → I
+  _∨_ : I → I → I
+  ~_ : I → I
 
-All the properties of de Morgan algebras hold definitionally. The ends
-of the interval ``i0`` and ``i1`` are the bottom and top elements, respectively
+All the properties of De Morgan algebras hold definitionally. The
+endpoints of the interval ``i0`` and ``i1`` are the bottom and top
+elements, respectively.
 
 .. code-block:: agda
 
+  module interval-equations (i j : I) where
+    data _≡_ (i : I) : I → Setω where
+      reflI : i ≡ i
+
+    infix 10 _≡_
+
```

Comment:
Isn’t it confusing to use `_≡_` for the inductive identity type, the same symbol being used for path types a few paragraphs below? I think you could simply list all the equations, with normal equal signs and without giving them a name, and simply say that `=` means definitional equality here. It doesn’t have to be valid Agda code.

Reply:
Yeah, I'll fix this